### DISCUSSION TITLE
Way to do difference merge with colab

### DISCUSSION THREAD

**Discussion by remybonnav** [TheLastBen/fast-stable-diffusion#1486] [Q&A]
I always get RAM overload when merging models with add difference in automatic1111 webui using google colab.
There is a trick to use the VRAM (discussed for the local version of webui I think) but I don't know how to do it with the google colab repo (And I cannot find again the thread where this was discussed).
Does anyone know how to do?

**Comment by TheLastBen** [comment]
It's a known bug with colab https://github.com/googlecolab/colabtools/issues I encourage users to open issues in google colab repo to force them to fix this "bug".

**Reply by remybonnav** [reply]
Thanks a lot.
